app/models/conversion_host.rb

```diff
@@ -241,20 +241,27 @@ def resource_supports_conversion_host
   end
 
   # Find the credentials for the associated resource. By default it will
-  # look for a v2v auth type. If that is not found, it will look for the
-  # authentication associated with the resource using ssh_keypair or default,
-  # in that order, as the authtype.
+  # look for a v2v auth type if no argument is passed in. If that is not found,
+  # it will look for the first associated authentication. If one still isn't
+  # found, then it will look for the authentication associated with the resource
+  # using the 'ssh_keypair' auth type, and finally 'default'.
   #
-  def find_credentials(msg = nil)
-    authentication = authentication_type('v2v') ||
-      resource.authentication_type('ssh_keypair') ||
-      resource.authentication_type('default')
+  def find_credentials(auth_type = nil)
+    authentication = authentication_type(auth_type) || authentication_type('v2v') || authentications.first
```

Comment:
Under what conditions would you need to pull the first authentication that isn't of any of these types?

Reply:
Wasn't sure about hosts and ipmi, or other types I might not be immediatley familiar with. If you think I can drop it safely than I will.

Reply:
What auth types are able to be added to a conversation host?

Reply:
Just check for the auth types that can be added by default not checking for anything

Reply:
@fdupont-redhat can it be anything other than 'v2v', 'ssh_keypair' or 'default'?

Reply:
Ok, updated.